modules/tensor_mechanics/test/tests/critical_time_step/tests

```diff
@@ -0,0 +1,38 @@
+[Tests]
+design = 'CriticalTimeStep.md'
+issues = '#13975'
+  [./crit_time_solid_uniform]
+    type = 'CSVDiff'
+    input = 'crit_time_solid_uniform.i'
+    csvdiff = 'crit_time_solid_uniform_out.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-5
+    requirement = "The CriticalTimeStep class shall correctly compute the critical time step for a solid with uniform properties."
+    design = "CriticalTimeStep.md"
+  [../]
+  [./crit_time_solid_variable]
+    type = 'CSVDiff'
+    input = 'crit_time_solid_variable.i'
+    csvdiff = 'crit_time_solid_variable_out.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-5
+    requirement = "The CriticalTimeStep class shall correctly compute the critical time step for a solid with variable properties."
```

Comment:
These two (this one and the one above) can be grouped into a single requirement with details to differentiate them. This cuts down on the requirement explosion and gives us a bulleted list of requirements with slight variations. See the requirement page for the syntax, or grep the the "tests" files in moose/test for "detail" for several examples.